### PR TITLE
fix nightly url

### PR DESCRIPTION
**Pull request by Ximinhan**
fix nightly pullspec url prefix

### CODE REVIEW

File: artbotlib/brew_list.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def list_component_data_for_release_tag(so, data_type, release_tag):
     so.say('Let me look into that. It may take a minute...')
 
     if 'nightly-' in release_tag:
-        repo_url = 'registry.svc.ci.openshift.org/ocp/release'
+        repo_url = 'registry.ci.openshift.org/ocp/release'
```
Comment (locriandev) [inline artbotlib/brew_list.py:146]: Can we make it a constant? It's already being used in many parts of the code, but hardcoded as a string

Comment (Ximinhan) [reply artbotlib/brew_list.py:146]: I only saw it also used once in rhcos lib, let's improve it later when we got more use case

Comment (Ximinhan) [reply artbotlib/brew_list.py]: updated in https://github.com/openshift-eng/art-bot/pull/177/commits/af075f51a57cabed5273b6a39d5e43d4d2953dce

File: artbotlib/constants.py [hunk 2]
```diff
@@ -35,6 +35,10 @@
 
 PROW_BASE_URL = 'https://prow.ci.openshift.org'
 
+NIGHTLY_REGISTRY = 'registry.ci.openshift.org/ocp/release'
```
Comment (locriandev) [inline artbotlib/constants.py:38]: We should use `registry.ci.openshift.org` instead, which is referenced in many other places, eg:
- pyartcd/constant.py:13
- doozerlib/cli/images_streams.py:596
and a few moew

Comment (Ximinhan) [reply artbotlib/constants.py:38]: this is art-bot repo

**Pull request merged**